### PR TITLE
RAU-78 make gem rails 3 and rails 4 compatible. 

### PR DESCRIPTION
**Pull request by ashwinisukale**
[RAU-78]

[RAU-78]: https://navigatingcancer.atlassian.net/browse/RAU-78?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by toppa** [comment]
Sorry to be picky with the tag, but can we make it `0.4.0-alpha`? (no `v` at the beginning and a hyphen in front of `alpha`). That will keep us consistent with [semantic versioning standards](https://semver.org/) which looks like what the NC team is doing with their recent tags.

**Review by toppa — APPROVED** [review]

**Review by toppa — APPROVED** [review]
Just one non-blocking comment

File: spec/action_view_extensions/tags_with_routes_spec.rb [hunk 1]
```diff
@@ -45,14 +45,14 @@ def to_param
 
     context 'animals index' do
       let (:polymorphic_array) { [:animals] }
-      it { should include('<a href="/animals"') }
```
[the remark this answers is not in the document]
Comment (ashwinisukale) [reply spec/action_view_extensions/tags_with_routes_spec.rb:48]: I have modified the regular expression for this, could you please have a look at it.

**Review by toppa — APPROVED** [review]

**Review by robertcornell-nc — APPROVED** [review]
This looks fine. There is an open conversation on spec/action_view_extensions/tags_with_routes_spec.rb and looks like some changes were made 4 days ago.  if that looks good to @toppa then this is approved.

**Comment by ashwinisukale** [comment]
@robertcornell-nc yeah, those changes are verified by @toppa

**Review by dbaratta — APPROVED** [review]

**Comment by ashwinisukale** [comment]
This PR is reviewed, hence I am merging to `Upgrade` branch

**Pull request merged**